docs/developer_guide.md

```diff
@@ -100,9 +111,9 @@ may wish to change the ports you map depending on your specific pipeline configu
 ```
 docker run \
 -p 21890:21890 \
--v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines.yaml \
--v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml \
-opensearch-data-prepper:1.2.0-SNAPSHOT
+-v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml \
```

Comment:
Similar to my other comment, it would be good to start to support configuring these values as directories instead of files.